app/js/components/SearchResults.jsx

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+import React from 'react';
+import classNames from 'classnames';
+import { LineChart } from 'react-d3';
+
+class SearchResults extends React.Component {
+
+  constructor() {
+    super();
+
+    this.renderStockInfo = this.renderStockInfo.bind(this);
+    this.renderStockGraph = this.renderStockGraph.bind(this);
+    this.renderResults = this.renderResults.bind(this);
+    this.renderNoStocksFound = this.renderNoStocksFound.bind(this);
+  }
+
+  renderStockInfo() {
+    return (
+      <div className="six wide column left aligned sr-stock-info">
+
+        <div className="ui header">
+          {this.props.stock.Name} ({this.props.stock.Symbol})
+        </div>
+
+        <div className="sr-current-price">
+          <span>Current price: ${this.props.stock.LastTradePriceOnly}</span>
+          <span className={classNames({
```

Comment:
className={this.stock.Change[0] === '+' ? 'sr-change-positive' : 'sr-change-negative'}
Why do we need classNames?